packages/ndla-forms/src/DropdownMenuItem.js

```diff
@@ -54,6 +55,22 @@ const StyledisSelected = styled.div`
   text-transform: uppercase;
 `;
 
+const StyledIsDisabled = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-right: ${spacing.small};
+  .c-icon {
+    margin-left: ${spacing.xsmall};
+  }
+  display: flex;
+  flex-direction: row;
```

Comment:
disse er det dobbelt opp av
```suggestion
```

Reply:
om du endrer den til column hopper det lille ikonet under i steden for å bli lite? 😅

Reply:
Da blir det alltid under da :<

Reply:
men trenger vel ikke begge? så skal vel ha column?